pkg/syncer/cnsoperator/manager/init.go

```diff
@@ -194,6 +184,18 @@ func InitCnsOperator(ctx context.Context, clusterFlavor cnstypes.CnsClusterFlavo
 					}
 				}
 			}()
+
+			if commonco.ContainerOrchestratorUtility.IsFSSEnabled(ctx, common.CnsUnregisterVolume) {
```

Comment:
Is this executed only on supervisor cluster?

Reply:
Yes it's inside the `if` block for initializing CRDs on Supervisor - https://github.com/kubernetes-sigs/vsphere-csi-driver/blob/master/pkg/syncer/cnsoperator/manager/init.go#L120